src/transaction.ts

```diff
@@ -457,6 +469,8 @@ export class Snapshot extends EventEmitter {
    *     PartitionReadRequest message used to create this partition_token.
    * @property {google.spanner.v1.RequestOptions} [requestOptions]
    *     Common options for this request.
+   * @property {object} [directedReadOptions]
```

Comment:
Customers will be using autogenerated code to build the directed read options. I think in nodejs we allow json object as input. But should this be marked also as `google.spanner.v1.IDirectedReadOptions` so that they know what type of class should be passed as input.
I am not sure of nodejs convention, but mentioning is as object could mean anything and is not clear.

Reply:
Done. Using `google.spanner.v1.IDirectedReadOptions` instead of custom object